src/test/java/com/ucsb/demonextjsspringtodoapp/services/Auth0MembershipServiceTests.java

```diff
@@ -26,56 +27,73 @@
   Auth0MembershipService service = new Auth0MembershipService();
 
   private DecodedJWT guestJWT = JWT.decode(
-      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTYiLCJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjIsImVtYWlsIjoidGVzdEBnbWFpbC5jb20ifQ.8yslpthMLMpwnlSctV5HN-fFJkKinpil61dJmw1m9Oc");
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lc3BhY2UiOnsiZW1haWwiOiJ0ZXN0QGdtYWlsLmNvbSJ9LCJzdWIiOiIxMjM0NTYiLCJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjJ9.6kcYK01GnVjCMELcgUyFJBYe1DeQ9y4NngDgYdwYwqE");
 
   private DecodedJWT memberJWT = JWT.decode(
-      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTYiLCJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjIsImVtYWlsIjoidGVzdEB1Y3NiLmVkdSJ9.paEMa69zK4AyN3PsNOGQsgovzexFzBKrR80Wa64TY7Y");
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lc3BhY2UiOnsiZW1haWwiOiJ0ZXN0QHVjc2IuZWR1In0sInN1YiI6IjEyMzQ1NiIsIm5hbWUiOiJKb2huIERvZSIsImlhdCI6MTUxNjIzOTAyMn0.RnoI4IH5bgp4uWd8VxKyVLrUTYu_JnUUhLpAaWc-0G4");
 
   private DecodedJWT defaultAdminJWT = JWT.decode(
-      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTYiLCJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjIsImVtYWlsIjoiYWRtaW5AdWNzYi5lZHUifQ.mpWDkt1IRqTMQaaYE7qqoZ280iaRyAMaRu_NqU6GZgk");
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lc3BhY2UiOnsiZW1haWwiOiJhZG1pbkB1Y3NiLmVkdSJ9LCJzdWIiOiIxMjM0NTYiLCJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjJ9.DhnSeuqo6YCsdb6qYV5AKsY_xrCZzZ6RHxnVP8WsiC0");
+
+  private AppUser exampleUser = new AppUser(1L, "test@ucsb.edu", "Test", "User");
 
   @BeforeEach
   public void setUp() {
     ReflectionTestUtils.setField(service, "memberHostedDomain", "ucsb.edu");
     ReflectionTestUtils.setField(service, "adminRepository", adminRepository);
+    ReflectionTestUtils.setField(service, "namespace", "namespace");
   }
 
   @Test
-  public void testGoogleMembershipService_isNotMemberOrAdmin_ifJWTIsNull() {
-    assertEquals(false, service.isAdmin(null));
-    assertEquals(false, service.isMember(null));
+  public void testAuth0MembershipService_isNotMemberOrAdmin_ifJWTIsNull() {
+    assertEquals(false, service.isAdmin((DecodedJWT) null));
+    assertEquals(false, service.isMember((DecodedJWT) null));
   }
 
   @Test
-  public void testGoogleMembershipService_isNotMemberOrAdmin_ifEmailNotInOrg() {
+  public void testAuth0MembershipService_isNotMemberOrAdmin_ifEmailNotInOrg() {
     assertEquals(false, service.isMember(guestJWT));
   }
 
   @Test
-  public void testGoogleMembershipService_isMember_ifEmailInOrg() {
+  public void testAuth0MembershipService_isMember_ifEmailInOrg() {
     assertEquals(true, service.isMember(memberJWT));
   }
 
```

Comment:
What happens if you pass in an adminJWT and call isMember?

Reply:
This is ambiguous... and we may need to define which behavior makes sense for the application.